nilearn/datasets/func.py

```diff
@@ -148,9 +148,10 @@ def fetch_haxby(data_dir=None, n_subjects=None, subjects=(2,),
     The anatomical image for subject 6 is unavailable.
     """
     if n_subjects is not None:
-        warnings.warn("The parameter 'n_subjects' is deprecated from 0.2.6 "
-                      "and will be removed in nilearn next release. Use "
-                      "parameter 'subjects' instead.")
+        warn_str = ("The parameter 'n_subjects' is deprecated from 0.2.6 and"
```

Comment:
You have accidentally removed white spaces between "and" and "will" in the warning message.